Description: Add support for TUN interfaces
Author: Roland Pallai <pallair@magex.hu>
Bug-Debian: http://bugs.debian.org/610334
Last-Update: 2016-03-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/base/pcap-snoop.c
+++ b/base/pcap-snoop.c
@@ -158,6 +158,10 @@ void pcap_cb(ptr,hdr,data)
           return;
         
         break;
+      case DLT_LINUX_SLL:
+        data+=16;
+        len-=16;
+        break;
     }
     network_process_packet(n,&hdr->ts,data,len);
   }
